src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHOT_TERM = 3;
```

Comment:
There is a typo in the constant declaration, it should be 'SHORT_TERM' instead of 'SHOT_TERM'.